pod/perlvar.pod

```diff
@@ -835,14 +828,14 @@ by this bit of code:
     sub show_n { print "\$1 is $1; \$2 is $2\n" }
 
     {
-    OUTER:
-	show_n() if $outer =~ m/$pattern/;
+        OUTER:
```

Comment:
This label seems to be intentionally outdented

Reply:
Fixed it.

Reply:
It should remain at the same level as the opening brace.

Reply:
Ah, I see. Restored.